<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quest_node</class>
 <widget class="QWidget" name="quest_node">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>1000</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Quest Node</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_box">
     <property name="title">
      <string>Quest Node</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="button_layout">
        <item>
         <widget class="QPushButton" name="move_node_left_bttn">
          <property name="text">
           <string>Move Left</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_node_bttn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="move_node_right_bttn">
          <property name="text">
           <string>Move Right</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="grid_layout">
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Next Quest</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QListWidget" name="giver_list">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Drag and drop characters from world object database.</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="isWrapping" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="name_lbl">
          <property name="text">
           <string>Quset Name</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="completer_lbl">
          <property name="text">
           <string>Quest Completer</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QListWidget" name="completer_list">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Drag and drop characters from world object database.</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="giver_lbl">
          <property name="text">
           <string>Quest Giver</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QListWidget" name="listWidget">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>64</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="name_entry">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabs">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="dialogue_tab">
         <attribute name="title">
          <string>Dialogue</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="start_lbl">
            <property name="text">
             <string>Quest Start</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="start_entry">
            <property name="placeholderText">
             <string>Quest Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="active_lbl">
            <property name="text">
             <string>Quest Active</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="active_entry">
            <property name="placeholderText">
             <string>Quest Active</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="completed_lbl">
            <property name="text">
             <string>Quest Completed</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="completed_entry">
            <property name="placeholderText">
             <string>Quest Completed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="delivered_lbl">
            <property name="text">
             <string>Quest Delivered</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="delivered_entry">
            <property name="placeholderText">
             <string>Quest Delivered</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="objective_tab">
         <attribute name="title">
          <string>Objectives</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="objective_list">
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Drag and drop from world object database.</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::AllEditTriggers</set>
            </property>
            <property name="tabKeyNavigation">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
